microsite/sidebars.js

```diff
@@ -27,6 +27,19 @@ module.exports = {
           'getting-started/homepage',
         ],
       },
+      {
+        type: 'category',
+        label: 'Deploying Backstage',
+        items: [
+          'deployment/index',
+          'deployment/scaling',
+          'deployment/docker',
+          'deployment/k8s',
+          'deployment/heroku',
+          'deployment/koyeb',
+          'deployment/flightcontrol',
```

Comment:
Do we want to remove any of the "community" ways to deploy Backstage? I assume Docker and K8s are the "officially" supported and maintained ones

Reply:
Hmmm, how do we feel about just removing them entirely from the docs site and rewording the overview page in the deployments section to point to [contrib/docs/tutorials](https://github.com/backstage/backstage/tree/master/contrib/docs/tutorials) which is essentially what you meant I think but without a tab 😅

Reply:
that works too!